metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2628,6 +2632,15 @@ def emit_datasource(
             datasource_info = datasource
 
         browse_path = self._get_project_browse_path_name(datasource)
+        if (
+            not is_embedded_ds
+            and self._get_published_datasource_project_luid(datasource) is None
+        ):
+            logger.info(
```

Comment:
just to be consistent with the other skip messages in this PR

```suggestion
            logger.warning(
```